<launch>
   <param name="use_sim_time" value="false" />

   <node pkg="datmo" type="datmo" name="datmo" output="screen">
      <param name="max_cluster_size" value="500" />
      <param name="euclidean_distance" value="10.0" />
      <param name="pub_markers" value="true" />
      <param name="in_topic" value="/detection/lidar_detector/objects" />
      <param name="out_topic" value="/detection/l_shaped/objects" />
   </node>

   <node pkg="detected_objects_visualizer" type="visualize_detected_objects" name="naive_shape_visualization"
      output="screen" ns="/detection/l_shaped" />

</launch>
